From 14c5a3727c697c8814d28f79053fdd773b6495d6 Mon Sep 17 00:00:00 2001
From: "E. Albiter" <ealbiter@gmail.com>
Date: Mon, 16 Sep 2024 15:30:21 -0600
Subject: [PATCH] Fix compilation with occt 7.8

Due to changes in the library names TKDEIGES and TKDESTEP
---
 ElmerGUI/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/ElmerGUI/CMakeLists.txt b/ElmerGUI/CMakeLists.txt
index 32c2b852cf..a4b21c0bc6 100644
--- a/ElmerGUI/CMakeLists.txt
+++ b/ElmerGUI/CMakeLists.txt
@@ -117,7 +117,7 @@ IF(WITH_OCC)
     MESSAGE(STATUS "  [ElmerGUI] OpenCASCADE_VERSION:       " ${OpenCASCADE_VERSION})
     MESSAGE(STATUS "  [ElmerGUI] OpenCASCADE_INCLUDE_DIR:   " ${OpenCASCADE_INCLUDE_DIR})
     # only link to available libraries
-    SET(ELMER_USED_OCC_LIBS "TKernel;TKMath;TKG3d;TKG2d;TKBRep;TKSTEP;TKMesh;TKIGES;TKG2d;TKGeomBase;TKBool;TKBO;TKGeomAlgo;TKPrim;TKShHealing;TKSTEP209;TKSTEPAttr;TKSTEPBase;TKTopAlgo;TKXSBase")
+    SET(ELMER_USED_OCC_LIBS "TKernel;TKMath;TKG3d;TKG2d;TKBRep;TKSTEP;TKMesh;TKIGES;TKDEIGES;TKG2d;TKGeomBase;TKBool;TKBO;TKGeomAlgo;TKPrim;TKShHealing;TKSTEP209;TKSTEPAttr;TKSTEPBase;TKDESTEP;TKTopAlgo;TKXSBase")
     SET(ELMER_AVAILABLE_OCC_LIBS "")
     FOREACH(_lib IN LISTS ELMER_USED_OCC_LIBS)
       LIST(FIND OpenCASCADE_LIBRARIES ${_lib} _index)
